shared_model/builders/protobuf/builder_templates/query_template.hpp

```diff
@@ -182,9 +182,14 @@ namespace shared_model {
       }
 
       auto getAccountDetail(
+          const size_t page_size,
```

Comment:
Why const value arguments? Maybe drop const? Same below for optionals

Reply:
okay, I see no consts in similar cases, so let's consider it a matter of codestyle. althouugh I see no drawbacks of making things const explicitly.